src/main/webapp/login.css

```diff
@@ -0,0 +1,14 @@
+body {
+  text-align: center;
+  margin: auto;
+}
+
+h1 {
+  text-decoration: underline;
+  width: 500px;
+}
+
+h2 {
+  text-decoration: underline;
+  width: 500px;
```

Comment:
Can you put CSS into src/webapp/assets and scripts into src/webapp/scripts or remove those directories?

Reply:
+1, put CSS into src/webapp/assets and scripts into src/webapp/scripts or remove those directories